tests/itemCounter.test.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @jest-environment jsdom
+ */
+import countLists from '../src/modules/itemCounter.js';
+
+describe('Count List Items', () => {
+  beforeAll(() => {
+    document.body.innerHTML = `
+    <div class="cards">
+            </div>
+    `;
+  });
+
+  test('countLists throws null error', () => {
+    const cards = document.querySelector('.cardss');
+
+    const result = () => countLists(cards);
+
+    expect(result).toThrow(TypeError);
+  });
+
+  test('countLists returns 0', () => {
+    const cards = document.querySelector('.cards');
+
+    const result = countLists(cards);
+
+    expect(result).toBe(0);
+  });
+
+  test('countLists returns 1', () => {
+    const cards = document.querySelector('.cards');
+    const paragraph = document.createElement('p');
+    cards.appendChild(paragraph);
```

Comment:
[OPTIONAL] Good job with the test, however, I think it would be better if you call the function that adds children to the parent container and check if the parent was appended.